pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -153,6 +171,11 @@ func (pl *CSILimits) filterAttachableVolumes(
 			continue
 		}
 
+		// ... the PVC for an ephemeral volume must be owned by the pod.
+		if ephemeral && !metav1.IsControlledBy(pvc, pod) {
```

Comment:
this could be checked in the switch case, right?

Reply:
It needs the `pvc` which is not yet available inside the switch statement.

Reply:
Makes sense, thanks for clarifying